spec/tests/ContentMetadata.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('content metadata spec', () => {
       }));
     });
 
-    it('set player type in custom tags', () => {
+    it('default playerType in custom tags', () => {
```

Comment:
Those tests names should be written like a sentence:
- `it sets the default playerType in custom tags` 
vs
- `it default playerType in custom tags`

```suggestion
    it('sets the default playerType in custom tags', () => {
```

Reply:
Done